### PR TITLE
Update timeseries/public example

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR updates the 'public' example of the `TimeSeries` object, mainly to include use of the GWOSC python API.

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
The CI is impacted by #1432.

File: examples/timeseries/public.py [hunk 1]
```diff
@@ -17,30 +18,48 @@
...
 
-"""Plotting public LIGO data
+"""Accessing and visualising public GW detector data
 
-I would like to study the gravitational wave strain time-series around the
-time of an interesting simulated signal during the last science run (S6).
-
-These data are public, so we can load them directly from the web.
+Data from the current generation gravitational wave detectors are published
+by |GWOSCl| and freely available to the public.
```
Comment (eagoetz) [inline examples/timeseries/public.py:24]: @duncanmmacleod is there a typo here: `GWOSCl`?

File: examples/timeseries/public.py [hunk 2]
```diff
@@ -17,30 +18,48 @@
...
+In this example we demonstrate how to identify times of a published
+GW detection event, and to download and visualise detector data.
 """
 
 __author__ = "Duncan Macleod <duncan.macleod@ligo.org>"
-__currentmodule__ = 'gwpy.timeseries'
+__currentmodule__ = "gwpy.timeseries"
+
+# Firstly, we can use the |gwosc-mod| Python package to query for the
```
Comment (eagoetz) [inline examples/timeseries/public.py:32]: I'm not familiar with `gwosc-mod`; did you mean `gwosc` here?

File: examples/timeseries/public.py (hunk 1 above)
Comment (duncanmmacleod) [reply examples/timeseries/public.py:24]: No, this is a substitution reference define in references.rst

https://github.com/gwpy/gwpy/blob/f9ce215df631c17485a085284780c56e7aa879b2/docs/references.rst#L103-L104

File: examples/timeseries/public.py (hunk 2 above)
Comment (duncanmmacleod) [reply examples/timeseries/public.py:32]: As above, this is a substitution reference defined in references.rst

https://github.com/gwpy/gwpy/blob/f9ce215df631c17485a085284780c56e7aa879b2/docs/references.rst#L18-L19

**Review by eagoetz — APPROVED** [review]
LGTM

**Pull request merged**